spring-session/build.gradle

```diff
@@ -25,8 +26,13 @@ dependencies {
 	integrationTestCompile "redis.clients:jedis:2.4.1",
 			"org.apache.commons:commons-pool2:2.2"
 
+	integrationTestRuntime "org.springframework.shell:spring-shell:1.0.0.RELEASE"
+
 	testCompile "junit:junit:$junitVersion",
-			'org.mockito:mockito-core:1.9.5',
+			'org.mockito:mockito-core:1.10.19',
+			"org.hamcrest:hamcrest-core:1.3",
```

Comment:
Can the Hamcrest dependencies be removed now that everything has been changed over to AssertJ?